sources/Application/Views/PhraseView.cpp

```diff
@@ -1129,6 +1130,14 @@ void PhraseView::DrawView() {
     DrawString(pos._x, pos._y, buffer, props);
     setTextProps(props, 2, j, true);
     pos._y++;
+		if (j==row_ && (col_ == 2 || col_ == 3)) {
```

Comment:
row_ and col_ indicate the current cursor position. Seems like we can infer where we are and locate what the command is in that position without repeating this statement for each command where it's written on the screen.